content/blog/2018-04-22-essurvey.md

```diff
@@ -0,0 +1,337 @@
+---
+slug: "essurvey"
+title: Download data from the European Social Survey
+package_version: 1.0.0
+authors:
+  - name: Jorge Cimentada
+    url: https://cimentadaj.github.io/
+date: 2018-01-05
+categories: blog
+topicid: 925
+output: md_document
+tags:
+- r
+- community
+- data
+- review
+- onboarding
+- package
+- data-extraction
+- social-science
+---
+
+
+<!-- # see for blogpost https://github.com/ropensci/roweb2#contributing-a-blog-post -->
+Introduction
+------------
+
+I never thought that I'd be programming software in my career. I started
+using R little over 2 years now and it's been one of the most important
+decisions in my career. Secluded in a small academic office with no one
+to discuss/interact about my new hobbie, I started searching the web for
+tutorials and packages. After getting to know how amazing and nurturing
+the R community is, it made me want to become a data scientist. So I set
+out to do it. Through out the journey I repeatedly found myself using
+the [European Social Survey](http://www.europeansocialsurvey.org/), a
+really neat dataset that collects informations on attitudes, beliefs and
+behaviour patterns of diverse populations in more than thirty European
+nations since 2002.
+
+After seeing a niche in the R package community, I created the package
+`essurvey` (previously `ess` in CRAN) to access this data easily from R.
+The package was accepted in CRAN in September 2017 and was well received
+among social scientists.
+
+Over a month ago I submitted the package to rOpensci, intimidated but

```

Comment:
To make the post more timeless, maybe say the month and year instead of "Over a month ago"